static/skin/viewer.js

```diff
@@ -294,8 +294,14 @@ function onClickEvent(e) {
   const iframeDocument = contentIframe.contentDocument;
   const target = matchingAncestorElement(e.target, iframeDocument, "a");
   if (target !== null && "href" in target) {
-    if ( isExternalUrl(target.href) ) {
-      const possiblyBlockedLink = blockLink(target.href);
+    // In case of wombat in the middle, wombat will rewrite the href value to the original url (external link)
+    // This is not what we want. Let's ask wombat to not rewrite href
+    const old_no_rewrite = target._no_rewrite;
+    target._no_rewrite = true;
+    const target_href = target.href;
+    target._no_rewrite = old_no_rewrite;
+    if (isExternalUrl(target_href)) {
+      const possiblyBlockedLink = blockLink(target_href);
```

Comment:
Please encapsulate (part of) this change in a helper function `get????Href()`. The hard part is figuring out what to put instead of `????` in the function name.